language-plutus-core/test/Normalization/Type.hs

```diff
@@ -16,6 +16,22 @@ import           Test.Tasty
 import           Test.Tasty.Hedgehog
 import           Test.Tasty.HUnit
 
+-- test [rec (lam dat (fun (type) (type)) [dat a])]
+test_catamorphism :: IO ()
```

Comment:
This test does not belong here. What you're testing is not a type normalizer. It's a normal form checker for types that got spelled wrongly when it was written.

Reply:
Well, I was initially testing both the normalizer and the checker.

Reply:
I've moved this to its own module since that's where the bug was.